person.rb

```diff
@@ -22,6 +24,11 @@ def correct_name
     @name
   end
 
+  def add_rental(date, book)
+    rental = Rental.new(date, self, book)
+    @rentals << rental
```

Comment:
Line 29 is unnecessary since the creation of the rental takes care of adding it to the person's `@rentals` list (`line 10` in the `rental.rb` file).

You should also check the order of the parameters you pass to `Rental.new`, the order should be as follows: date, book, person. So on line 28, you should switch the order of the parameters like this: 
```
Rental.new(date, book, self)
```